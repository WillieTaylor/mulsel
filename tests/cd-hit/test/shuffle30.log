================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.90.out -c
         0.9

Started: Fri Dec 11 18:04:29 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 82M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 89654694

comparing sequences from          0  to      30013
....................    10000  finished        661  clusters
....................    20000  finished        963  clusters
....................    30000  finished       1152  clusters

    30013  finished       1152  clusters

Apprixmated maximum memory consumption: 83M
writing new database
writing clustering information
program completed !

Total CPU time 0.68
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.80.out -c
         0.8

Started: Fri Dec 11 18:04:29 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 82M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 89654694

comparing sequences from          0  to      30013
....................    10000  finished        360  clusters
....................    20000  finished        507  clusters
....................    30000  finished        588  clusters

    30013  finished        588  clusters

Apprixmated maximum memory consumption: 83M
writing new database
writing clustering information
program completed !

Total CPU time 0.74
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.70.out -c
         0.7

Started: Fri Dec 11 18:04:30 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 82M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 89654694

comparing sequences from          0  to      30013
....................    10000  finished        244  clusters
....................    20000  finished        338  clusters
....................    30000  finished        377  clusters

    30013  finished        377  clusters

Apprixmated maximum memory consumption: 82M
writing new database
writing clustering information
program completed !

Total CPU time 0.74
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.60.out -c
         0.6 -n 4

Started: Fri Dec 11 18:04:31 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 3M = 3M
Miscellaneous   : 0M
Total           : 20M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 97433934

comparing sequences from          0  to      30013
....................    10000  finished        172  clusters
....................    20000  finished        228  clusters
....................    30000  finished        253  clusters

    30013  finished        253  clusters

Apprixmated maximum memory consumption: 20M
writing new database
writing clustering information
program completed !

Total CPU time 0.95
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.50.out -c
         0.5 -n 3

Started: Fri Dec 11 18:04:32 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 17M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 97804374

comparing sequences from          0  to      30013
....................    10000  finished        129  clusters
....................    20000  finished        164  clusters
....................    30000  finished        173  clusters

    30013  finished        173  clusters

Apprixmated maximum memory consumption: 17M
writing new database
writing clustering information
program completed !

Total CPU time 1.19
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle30.fasta -o shuffle30.40.out -c
         0.4 -n 2

Started: Fri Dec 11 18:04:33 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 30013
longest and shortest : 90 and 56
Total letters: 2232323
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 5M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 17M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 97822014

comparing sequences from          0  to      30013
....................    10000  finished         90  clusters
....................    20000  finished        103  clusters
....................    30000  finished        105  clusters

    30013  finished        105  clusters

Apprixmated maximum memory consumption: 17M
writing new database
writing clustering information
program completed !

Total CPU time 1.34
