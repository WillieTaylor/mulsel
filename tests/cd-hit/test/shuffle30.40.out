>seqG1614 = G family seq 1614  0.000706 35 G
CQGLGGGRGGHLQRGGGLCTCGGSPNGHVLLGGQGGRGGPGTKQWGGSGCGFGGELGSMGVHWGSGIQLGTHMGLGGERQPGG
>seqL1459 = L family seq 1459  0.001713 34 L
KLFLILLGINTMSPYTELMELLILLLEILLYINGLLLLTLSTAGPLLDYTHELYMCFLLLPLAVLGLLLDLMMHLWLTLLL
>seqV1698 = V family seq 1698  0.001752 34 V
YVLVLVEVVVERVVCVVVTVKLLCVNVLQDARFWMNVAAVAVNEVVPWVNVVGVPYQVPVVLVVHMWVHNNVVVKHVVCA
>seqW1162 = W family seq 1162  0.002285 35 W
RLWWSWTGWWYWSIFPWFWRPWLNWWWWFVWSECWDWKWLEYWKGWWDWWCMNGWPPIWWGMWCWDLEVGTWWDLWKMWWMWWLWRH
>seqS1764 = S family seq 1764  0.004374 34 S
DSNSFASSVNGSESVCSMHEMSSARLSAESSVISSSMVPSYFSNSSLQSSSSSMDKEWVLSSVQQNSSLHSSSAQSPVAQSQSQIDK
>seqD760 = D family seq 760  0.009265 34 D
NMGDSTRNDRDDINDDDTDGVDDDNCACDDKDWDDGDDEDNDVWDCDPDDDDSNDPLDGDCDDDLMRRCGDDTCCQ
>seqM1938 = M family seq 1938  0.010567 34 M
KRMMMTMVQMDSMRMAMAMGQMQAAMFVMMMMKIMMMMMEMMHIWMMGMSGITNQVSFMDKGSHMMFMVTWMMPTMMNQMRNV
>seqP1013 = P family seq 1013  0.011700 34 P
PNPPPYYPYPKQPNSDHVPPKPPAPASPPSLPSKSSFPPKESILPTPKHMPPIHPDFPSPPDPSPPLSWTNIPPPDMKNPNPP
>seqI1574 = I family seq 1574  0.012585 35 I
VIYFAINRDMIIYGMMSYMIIISIDIWLIIVICCIIQIIIIMIRDLIIIMDIFAIDIIYWIRHQILIIIRIILSYII
>seqV942 = V family seq 942  0.014115 34 V
VRHDVQVVVVHVTVVMVSTVIYQRPLSVDLVHPNNVVLFHVTVVVVVWHVVIWVVHNYVVVGVVKDTHPTVSCVRIPDVV
>seqI825 = I family seq 825  0.014425 34 I
HYVAWHIIAIIYIIMIDKEKIIDMIQIRICIIRHHWIIIIIIIIQIAADALQPIPGDISHAIGIIIIIIIIWELVHLC
>seqC754 = C family seq 754  0.019071 34 C
AECFTNCIPMPTCCCCCCTCCPMWITERCMWCSCEMRCCGICCCAFWGCCAFCVCCCCHCIFCCCSETSCCCWCCL
>seqI1340 = I family seq 1340  0.019111 34 I
GIRIVAIQINCIMISMIMNIMEMFIAYIIIFWILIHIKSCIIIMTIHIICADISIIAFDIFHDIIMIEIIISLQIDDIESNDICH
>seqN536 = N family seq 536  0.019276 35 N
FMSMCSSNNKNNHRPINGANNNMNNMQGNNANRNNCNNNTIMNHKNHHVANNNNNGITNKNNNNECPNNNQNNSDGT
>seqS898 = S family seq 898  0.020769 34 S
WDSWWSVISCRSASSFKSASSLSWCMLSDSSHKSSSQPSSRQCSSRMWTSFFYGSSMPNSSKKSNVSNSSSSPSWAGSSQH
>seqE864 = E family seq 864  0.020904 34 E
EFEEEDGEEEPEEECLEEEMETNEADYFDEWNWQCELIETSEEERYEETREEKEPVEMALEECKEEEPEMPWEFSR
>seqC1029 = C family seq 1029  0.021252 34 C
GNCCNFQCNICTMPTCYFHRWCPCICTPGDCGCENWWNMCCCCCCFWCCHGACICCCCIHNCCCCICTEECCCGCCCL
>seqK895 = K family seq 895  0.022755 34 K
WKKKKYHGCKLNKFKKKGIKKKKKGKCKMKKKTVGTQKLCIKNKADYIAQEPKSKHKKPTPIKKKKSCKKKMFYKA
>seqT903 = T family seq 903  0.025894 35 T
STQACTWDVTHMKTTPNTTPGTTQPWTTNKTTETRTTTTTDHFTIKTKYAATTRSTTTLTTTETTYYTTATQVRTIR
>seqY846 = Y family seq 846  0.031491 34 Y
YAGYCRAYIMIEYTYYYWYGPVYYYYHYIYYAYYARYYYTGPYLNYYVNYFYWMYYVYCCCMYEMSYSYYAYYLWWQ
>seqV590 = V family seq 590  0.031719 34 V
VVAIVQHCVAMETVFVQVVVSERVINVVVTVVVVMVVVWVQVQQVVKVFVVVQKVIKVTVVYNTVVRPGR
>seqD1520 = D family seq 1520  0.032019 34 D
APPHDTTETYYDTCDQDLDQDDDDDYDADHGDDTNDFDDDDYEDLDQLDKTAVFDQNGGLKWRADYCWDLDDKEDDDDDKADIRPDV
>seqE1836 = E family seq 1836  0.032869 34 E
HERAYEAVEEFTNGEEEEFGEPEPIETAFQDGEEGEKEEPTETAEEQEPFEEEHPEPEMRQTREDGEEEAESSEEMPYEERR
>seqK284 = K family seq 284  0.035067 61 K
WVWRKYKKKTNLKKKYKKKKKKKKPKKKKKKKKKKKKKKKKKKKKKLKKMKKKKKHKKYEKKLGKKKKEKKKKKSKKKKKNA
>seqS1118 = S family seq 1118  0.035620 34 S
FVSWWSSISRSCAWSQCCSSGASFYSQESMDSMSSSSFHKSWEQCGSCCGSRRSELEEQSSSSFSSSSATKSFQSWHSFTQYTSWSSFC
>seqQ1165 = Q family seq 1165  0.037351 34 Q
KQQYQCEQPQPYPVPQQTQGFQLYQVQWQGQQCLTERTCTEQQTYRQQLQHPNVQDMIVQQVQKQGQHQMPFWMFDQMQYQLQQQQQ
>seqA605 = A family seq 605  0.037937 38 A
SSCWDKATAAEFAAKIAQHAATKAAAWGAFDRAWAAAFAADLAAAAAAVAAAAAAAAQSSYAAKAAFAACWGK
>seqH1898 = H family seq 1898  0.039047 36 H
KHIFHHHLHQHDKDHMASEFCHHHMHDKHIWHNEHHWCVDQHHHHHNHKIVHHHCHHHAHITDKRQHHSVEKKEEMHHQMHHKERVRHHI
>seqR841 = R family seq 841  0.042202 35 R
SHRRRRSEQRRCRQRRTVRCRDAHATHRYRARMWPWSMKIRNRRTQRQRRNRRARRVWTMHMCRKGRRRLRRKDRWQRFRLCCGRRK
>seqL1631 = L family seq 1631  0.048124 34 L
CLVQVIVTLLVRHLGLLKYLLLLILHYLLNLCLTLQLSGCSLSLFHGYHMLLPCLLLLLHYLLLMMLWHACLLLL
>seqM1412 = M family seq 1412  0.048363 34 M
MMMDIHHMMMMMMSLRQMNISMSMFMMWCSEMWMEDGMCMEGMNTMQCLMQKMMMDMMVKPNMPADPNMSMMMPGANMVYMML
>seqG1947 = G family seq 1947  0.050773 34 G
RMQGWAKIHCGNGGASGIVIDDVGGQGKCGGGGHGGAGLGDGHQGKWGCCGGACGEGDDGIIGGAWSMGGGGGVGFSGG
>seqI1145 = I family seq 1145  0.050899 35 I
GIVAIQINAWRLINYCSANDNEIIFIIKIIIDIIHIKFSVIHIIKIIIIIADRIPIIFDDKGHIIIGEPIQDIIWMIIIKPIC
>seqQ881 = Q family seq 881  0.055802 34 Q
KEFDEQCQQQPQQFQWQYGFQKQIQQLTGAQQQLAPQQNFTQTSYKVQQKKQFFYKGQQQVYKQQQTDMQNDQQQQFQQ
>seqT1883 = T family seq 1883  0.065911 34 T
TTGCTTFTTQTTTETYTTTCCEQITACSQTTTPTTPMTTTEGTITDRTTPTWIFGCNTTEDEITEPYICTTHCQTQSTQCK
>seqP762 = P family seq 762  0.067435 34 P
IMYFHPPNQGSWYPPYTPPNVPGPPAASKSFPPYFQSPRPPDPHMEPPYPQFSPPPPPDPPPPPPPQMPPMPHPN
>seqV1987 = V family seq 1987  0.069498 34 V
YVLYICPVVVVMVVGTVEVVFLCMAVLVVVEQSVRVPVVGCVNCDVMFVMNSAVLVSVPSKVNHVVVCWVPVYDPVCCVVMV
>seqF1891 = F family seq 1891  0.077272 34 F
FECWCKIFEDFKDFFQRFYFFAYCFFFKCFEFFFPTWFEADCEFFFFYDGFALVFCFACFDWFCIQGCFFYFDFYDNFFFFFKQDDWE
>seqE1490 = E family seq 1490  0.078609 35 E
REEWFWETTREEEEQEEYREGETFYVREEWELSKEVNEEERYTESEEERREEEETEKTEPEWAFVELDSEAEEEVDKE
>seqA1924 = A family seq 1924  0.083499 34 A
AAEAAACAAAAEATFRAVFLRAAAAGARAEATLWAMDEATADSCVRAFSFRFIAAARCSSDAMHACFSEERIAASARAIAAA
>seqT1530 = T family seq 1530  0.085559 34 T
YTTQNTTDFYSTTTPYDFTGTITTPTTGQETRNYTTEACTTITWSTNTLATTWTMQSTTTPRTTGTTNTTCSSCRGNPTR
>seqW696 = W family seq 696  0.085867 34 W
EQWTYGWFKWKRMWNWIWPPWWYWFWYWNWWYKCSEWALWWWYWWAWCLHCGWQIWWPIWWEWMWWFWWWSIAWWSFWKMG
>seqS1907 = S family seq 1907  0.089985 34 S
ISSWHSSSHASISSRSQSECHIRLEESCSISLSSMPVEISRFLNSSQHHSRRSSFSSSWSVWSSSLLSSKSARPPVAQSSEQLSE
>seqM1682 = M family seq 1682  0.091202 34 M
MMFFMHMHMTQTMMMANMIMRITMMMMEFLFMMVMMMEHVFCDSMMTYMMHTQPPMMMVRVMTFMDSPVQMVMIVPTMMENFVYRMM
>seqR1260 = R family seq 1260  0.092013 34 R
ERRRWRRRRAQRINRRVEHRARWERCKRRRKRKQNRCLLARAVPARRICRRGRRRQCRRRRDPNRRRDYVNHRL
>seqI1827 = I family seq 1827  0.094250 34 I
PYNICICQDIQIIIKMYYYGCSIIIIKRCCISICCIIILVIHIAIDGTIWICIIIKNIHSIKIHNYCNIGIIISVIIFIWIC
>seqC1424 = C family seq 1424  0.097177 34 C
CCNVLRCQCYCQCDCFCCDCHCCCQEPSCRTVTDCSCGCCCDQCAHCCLCCCCLESILLACCWCICCHREQCASTRSEICC
>seqQ1928 = Q family seq 1928  0.103808 34 Q
QYGQQPWFNQQQGAFKELQDGKQPQQQFENDQFYKTTTRCYKQQWQQKNQYQQQIRQQQFNIQQNQVRQQWQHGQQIQQPAQGP
>seqD1971 = D family seq 1971  0.104303 34 D
PDKWDDFDPDDIDDDMDYDMDAYMVCDIYSDDIKKDDDHKDNDAIVDFDDYMIKDDMDNDDADDMSTDKNDKPPTDCICYD
>seqL1187 = L family seq 1187  0.116302 34 L
LLLLLLLLRNPVLFYYLMELVLLLLEQLSNRLIMYFNHRMLGVLSLILTLTLSLHLPLLWLEKPLLGTYALWHLWALL
>seqN1250 = N family seq 1250  0.116479 35 N
NGKENNDWSNRYDGNNNPDENCENNAFCGNCVNPENNWNNNENNNNNNYEDWNCTELNWWNNIMNNNMNDNENMNFTWNA
>seqM402 = M family seq 402  0.126502 46 M
PNMTFMLMMMAMMLLSVTVMMRMMPMNMWHMMMMMMCMLMMLCSMMMMMPMMMMMMSMMMMSMDMMMMMMMM
>seqK1654 = K family seq 1654  0.127899 34 K
CDKSFKKKKKKADFKSKKKIIHGWLHCHKNKENKHPAGKFFSSKKKWPQKWHVKGKKKSKKSFKPKVNSKAMKKKKKK
>seqQ559 = Q family seq 559  0.138038 39 Q
KQEMQMQQQQQQMQYGDQQHFQASQQQSQAHYQLQQKQTQQYVQQEGQQFGQQQQNMQNQQYLCNIQSQQQQWQC
>seqN1761 = N family seq 1761  0.144040 34 N
NPENWNNINQNERYHINDPNSHCNWLNNDQFNNWKNANNITNVNNPMRNANYNLNSQDNLCNIYNVNNHNNINHWNENPPPVNV
>seqA1264 = A family seq 1264  0.149147 34 A
TAAAKAAARAKHAYFQAQRMERARAPAAFAETAQRWMAARTIRLACQATAAEALAAYAMAAYECAYAKTYTLARQCCAWACGAAC
>seqP1965 = P family seq 1965  0.169353 35 P
VPPFYLPYPPFPFPPNPPPVPFPPPWQRPFPCCPNMRFVMWNFQVPPTCPVKHMMMWNPPPPPPTEPCHYVPPPPVDPNPVP
>seqF833 = F family seq 833  0.172025 34 F
CVIDNREEFPFVLFFNFFFQWFFSFFPFHFFFFYRTEPFQKKVSKLLVDVYDFFFTFFFFMGFFAQNNFFFFIFFIPFPF
>seqA1530 = A family seq 1530  0.173723 35 A
RAYEVFALEANHAAAFTACATLNPAPAAAETQAWDALAATPMHAAAEARPLANMPACAAMYIALVVAAAAAPAIANAAA
>seqC1652 = C family seq 1652  0.181071 35 C
SQCCCCCLDCCHCCCCCVHCCCNWYYKCSCWCCECCNPKKRCHRQYCCKCTWSSECCSYCDGQKDVCKLPCCCVMPGFCECPC
>seqG1062 = G family seq 1062  0.184521 34 G
HGGLKWIIGGRGGEGGKSILGGGESGGEGGYYGSDVGTIGGTMPVGEYTIGIHKGHGGGFDGNNHRWGSGVGGGHPGDIDPTGGM
>seqM932 = M family seq 932  0.186403 34 M
SMNIHNMMMFHMHKMMIMMDYVQKMMMRMMSMMMFMLMVHMMNMFFLIMVSWCKNMMMMRMMLTMDEMMWWAVDCPMRHGYPM
>seqC1852 = C family seq 1852  0.186858 34 C
WNNSWTCCCCAECICDCCCKNGVCCCICVCAKRVYVCCCELVCCKCCECICCVNCCCCCCYAGCIILCRCQCFLEKAL
>seqL859 = L family seq 859  0.193328 35 L
TDVALGLFENLEYIWYNDLLLNHLLLSLHLKRATCFLLHLLLNNLLHLSHDTGLLLSLRLLLLLILVLLLTWATDIMLLHL
>seqH1641 = H family seq 1641  0.196363 34 H
HHNSQHHCHRHNMHIMHHWCFGHHHHEHQNHNHQHNMHHAQMHVHILADHHQLLKNFEHHHLHCHHKHHPHDFKWHHWNPT
>seqM1840 = M family seq 1840  0.201804 34 M
KFMMMMVMDYMMMMEMMMMDMMQMFVQMMMHMMEVGPEMHMDGMMMTLQVSMKGSMESPQMMVMMPTMSFRMW
>seqP1324 = P family seq 1324  0.220934 34 P
DNCPRPNYEPIPGHVRSPPEPCPWQPPCSIPPPSQPPVPPPKPHQHHTRGLGPPIPVPPPQESPKPCVPLSQNPQKPWWPRKPHWPP
>seqP1566 = P family seq 1566  0.222846 34 P
PDPPRPPWPPDNPPVAPPDPPWPSPPPPHYIPLAPPYPNTHPTQNPPVVIHDKPGTLPPSVNQTNQPNRTPPPRVPVWPL
>seqA1000 = A family seq 1000  0.239796 34 A
DAAALAKIRDKAAAAAAEAVAAMMYEFAPEAFAIFAFARVMPAAIFLARRFAEAVKDQVAQAEAAACQWAKAPAAQDAWLGAYK
>seqS2009 = S family seq 2009  0.241892 34 S
SCSHSSSDQSVCSRSSSSHSHRASEQICSILISVSSEVEFECSSSSCSILSLSSFSYWVSSEMDSLESKSHYPPVQSSDLSM
>seqN1928 = N family seq 1928  0.246342 34 N
ENVNNLHNNNNHMINIPNFNDNKNRHDNMNNNNLRKMVNSNMNNNNYKLLNSQNDHNIYNDENNNSGNNVNNV
>seqT1166 = T family seq 1166  0.246392 34 T
SQGLMGTTTFTITMTTTTTWMTTCQMSQTHTTTPMTELTHTTTDAVIKPTYTWWTTTWSTMTVETNTTYCPQMQVRTLTTR
>seqA1788 = A family seq 1788  0.249717 35 A
TAEDAAAAAPAAAFCRALYAAEAAGTAQLWSWDARPNGAAAAIYMAQAMAAEEMMAMAACEEAAIARAAAYIWAA
>seqV1226 = V family seq 1226  0.255188 34 V
MMVCDVATHVVVVCTVVTFVMVVWFTVLCVVVYIVVVQMFTVVDVVNVVNVVMVVVVGYGFNFVNPATANMIEVVVI
>seqE1973 = E family seq 1973  0.283797 35 E
LERACEEEEPFTANQFWCEEAEEIESEAKDEEEEEEQECIQEANAQEVISEENEEPLWQEEFEGCEEIEEEEQEPEINRNR
>seqN989 = N family seq 989  0.290649 34 N
QKYPTPANNNSNGNNNEAECNENNAGNHNVNKNPNNLNQNGNPNKVHYVNGHNINNGNWQNNNEGNCVNDENNVSCFNNMA
>seqR1474 = R family seq 1474  0.323543 34 R
RGRQERRRGNRRGINRRRRERWAGKNRERRGLRKRKMRRRGRRAKHKPREYGIRRRRRSRRQDEERTGRYVRGR
>seqT1720 = T family seq 1720  0.343651 34 T
FTGTTIGTTMNNQSTTLYNAWTEETQITAMWTTTWKTYTTKIKCETTIPERTTTWTWPTGQSTTSTTQEWMVTNETATTTTQTQA
>seqE1283 = E family seq 1283  0.351424 34 E
EEEEEELEDICEEEEENEFVFEEWLCEEMECKVEETWYEEYAIGLYESEEVMELEENPEVVEGEEEPAMMHWE
>seqF1586 = F family seq 1586  0.356494 34 F
YTFFNFQDFDMQFFPGFLFFYTDQFFKQTFFEAFDFTGDTHFFRHEVFEFHLCFYFFYFFFHMFFFDGYFFFDFAFFHDDIH
>seqW1844 = W family seq 1844  0.361530 34 W
WLQGKQWWLVDDWWANWDLDYWNWYIWWWWWWLVYWCIVLKWKWWWQYLWLGWMCMNNWIWLADWWLQWWGWMWWAVWGGWWVPVW
>seqY1759 = Y family seq 1759  0.371438 34 Y
TDYIVYYYGNYIFVYFYYYHYYYDYFYRYLYYQAYNIPSDYTYLCDRFYRDSYDYLRCYIYYYSDDYDDMYYALYYKYLQRADYV
>seqY1071 = Y family seq 1071  0.377755 34 Y
YAYWYRYTQAYYRYGYYTHNYSAKYQQDYYYWGTEYYEFTAYHWDTGYYYWNAYSYYYEYWCHYVYECCKYTMFKSYWYAYYYMYPQ
>seqG846 = G family seq 846  0.382465 35 G
KKGTKPIGGGGGELKGANGGPGLGLEGVSGGCGHNGGGTMGGGGGTWGKWNKQGGEWQYDNGRDNGGVGKGGGGNNCQLGH
>seqW1485 = W family seq 1485  0.401892 35 W
WWWDVTVWSWWADWFWHWWYQWVLNKHQALWWQNWNWQRWWWWYRSWMWGWCWWPQLWWWWDWVGWWGWHWGWWLT
>seqL1959 = L family seq 1959  0.407956 34 L
LPNMPLLNLHINVLNESLMCLLMMLDLLLGLIESLLLRLLLMLNLTGLGLLYLVLQGFSPLPLARTTLDPSLSLLIEAMLHWLA
>seqF1363 = F family seq 1363  0.435433 36 F
FEIYFEFPFPFNNFSFTFETCFQFGPMMFQNFEFGPVFFFFCFFFVFFFNFFTIGFHFFVFLFFHMPMFFFDAAGFWMCPFG
>seqG1851 = G family seq 1851  0.450055 34 G
RFGWAKIPIGNLGAGKNIPVGGGSVCGIGCGGGGEGGQLGEGMGGGGIGDGGGGGIGYGGGGVLFFGFG
>seqD1860 = D family seq 1860  0.456522 35 D
PDKDWQDDFSDDIDFDMATDDDHGADSDSIDDDDDDNDTDVFPDDDNNDTNDNVADDCDDVNSVGNDDVDETPDCFDA
>seqR1790 = R family seq 1790  0.470126 35 R
RWERCWRRRIRRTRQRRKNLKRRRKFRMRKVPFRWRERRAKLGSHARREKRIKRRVRRMYHNDRRRRVNRHRVTIRVQSRMRHPV
>seqR1915 = R family seq 1915  0.475727 34 R
RNARCWSTRRQTSQRRNLVRRRQRQRRRIRTRRRRRERSRGLGRTASVESAIKRRVRRHYANHHCRCRRRRTLSIRVRQAWR
>seqS1471 = S family seq 1471  0.494520 35 S
DSSYSSSNAPYPLVCVCSSDSSFYSSEPSSVSSTDSASYSSCSRSSQSSDYGCDSWVSQSLHTIDTKSSEYVLMSSTSQSSMSKLK
>seqY1991 = Y family seq 1991  0.536700 35 Y
DCHVIFFYYQYKYVVCYYKIYICAWLPYGYHYYWNAYYISYNGKGKDCYFYYTMQYYYMYYHYVYYYNDYCQKYYYYNNYYYWQYMRV
>seqY1575 = Y family seq 1575  0.547921 34 Y
YMYQTYVMVSTSVMYHNMFYYSSYPYYMNHYMYYRMYWYRYHIYYMYQPYYYYNDYYDDVYYMYYMYSMYQYYYY
>seqT1363 = T family seq 1363  0.559250 34 T
TQVTQWMTFNTTTTTDTGGTVTMGTYHYTKSTNVTTTETNTTYTTTTGVPNTWNTNMSTTTTNVNYCPVTVRGTTTR
>seqV1516 = V family seq 1516  0.603963 34 V
VPVQDVWVNEVCCVNVVVKVVYVVFENVVVCFVFNIQCHVRVVQYVVHVVVVVGVWVYQDFHTVWGTVNFCVKNKVVI
>seqQ1760 = Q family seq 1760  0.618018 35 Q
YHQQQPWHHGQQGQVQELQNQQQVQPQKQEMQQPQQTDRYQQWFTIVYQYPQDQTPQCQQQMQQQQTDQIQPQQRWLG
>seqG1450 = G family seq 1450  0.722673 35 G
CGWGCLYGGGGHDQGFEGLGCGLFHGKGLGLYGGGTGEPGPKEWGGCGGGGNEGGEQGGGMHGYQGKGRCIGGEKGQM
>seqI1938 = I family seq 1938  0.743590 34 I
IWITVIIIHYIWHTIIMYISILSDKRIIVICIISCIHIIIKIIKEEMWWIIIINICRIKERFIPINIIIASVEIFTWIR
>seqK1851 = K family seq 1851  0.829044 34 K
KVDNKFLVKEVDKFKFHKKIKKADWKHKKWNRKPCFNGVMDAKKKKNNKWPYQMKKNKKACKKKDEVKSITVVKKDNKVAKKKHKK
>seqY1310 = Y family seq 1310  0.904492 34 Y
NKNYHEVAYFYYYRNYYKAYPYQYYYKYWENHEYTAQYWWYGYYYYNRPNSYMYQWYRYGCKQDWGRYEYYNYYYYYYY
>seqH2009 = H family seq 2009  0.929778 34 H
TIHFLGHHLQHHVFLMHECHSHGHHHAHEHHHHHPHVFGHNGTHHHHHCSHQYLHFDPRRYHSHEHEFHHHKLHPAVRTGTHI
>seqH1235 = H family seq 1235  0.943386 34 H
HHECTAGFHGHPAKHHTSTHHHNHHDQGHPGGHAHHTTRHYFDRHSVFHVHAHQHHHHIFFFLGHHHTHHWLHHAPHHHWSE
>seqQ1547 = Q family seq 1547  0.962155 35 Q
WPFQQQMVMPWGQFCQELGNQKQQQQWAQQDEIQDCQRRHCTDQQQQQQWQPNQQIQDQDLQIDQQMQNQFHDQMQQQQGV
>seqD1214 = D family seq 1214  0.976998 34 D
DMDDDTRLYRDEDRQEQYLDADWDDDLDDDDDGDETADEGDDLADDQDDKNAEDWDEHMDPLMPFQKGDEDDDHDIKCMVHDCVD
